ui/app/templates/components/form-field.hbs

```diff
@@ -138,6 +138,7 @@
     @value={{or (get model valuePath) attr.options.defaultValue}}
     @placeholder=""
     @allowCopy="true"
+    @onChange={{action (action "setAndBroadcast" valuePath)}}
```

Comment:
what is the difference between the inner `action` (inside the parentheses) and outer action (inside the curly braces) here?

Reply:
So the outer one calls a curried version returned by the inner one, onChange just passes the value, but the inner one returns a curried setAndBroadcast with path already set.